### PR TITLE
docs: specify `timeout` unit for `resty.kong.log.get_log_level()`

### PR DESCRIPTION
**Pull request by chobits**
Reference the effect [here](https://github.com/Kong/lua-kong-nginx-module/tree/doc/new_get_log_level?tab=readme-ov-file#restykonglogget_log_level).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -471,8 +471,7 @@ resty.kong.log.get\_log\_level
 
 Returns the current dynamic log level, remaining timeout and the original log level.
 
-If the dynamic log level is not set or not active,
-`timeout` will be `0`.
+`timeout` is in whole seconds, as an integer. If the dynamic log level is not set or not active, `timeout` will be `0`.
```
Comment (ADD-SP) [inline README.md:474]:
``` suggestion
Returns the current dynamic log level, remaining timeout (in seconds), and the original log level.

If the dynamic log level is not set or not active,
the `timeout` will be `0`.
```

Comment (chobits) [reply README.md:474]: image

hmm.

Comment (chobits) [reply README.md]: fixed in new commit

**Review by ADD-SP — APPROVED** [review]

**Review by chronolaw — APPROVED** [review]

**Comment by chobits** [comment]
@ADD-SP @dndx I think it could be merge now.

**Pull request merged**